internal/util/compressor/compressor_test.go

```diff
@@ -146,7 +146,7 @@ func TestCurrencyGlobalMethods(t *testing.T) {
 
 			data := prefix + fmt.Sprintf(": %d-th goroutine", idx)
 
-			err := ZstdCompress(strings.NewReader(data), compressed, zstd.WithEncoderLevel(zstd.EncoderLevelFromZstd(idx)))
```

Comment:
The purpose of this change is:
Run this single test on a machine with a large number of cores. If the WithEncoderLevel of zstd is too large, there will be a chance of Panic.